onboardingtaskapp.client/src/components/Product/Edit.jsx

```diff
@@ -1,33 +1,49 @@
 import { React, useState, useEffect } from 'react';
-import { Modal, Button } from 'semantic-ui-react';
+import { Modal, Button, Message } from 'semantic-ui-react';
 
-export default function Edit({ item }) {
+export default function Edit({ item, isUpdated }) {
 
+    // states
     const [open, setOpen] = useState(false);
-    const handleOpen = () => setOpen(true);
-    const handleClose = () => setOpen(false);
+    const [message, setMessage] = useState(null);   
 
-    const [formData, setFormData] = useState({        
+    const [formData, setFormData] = useState({
         name: '',
-        price: '',  
+        price: '',
     });
 
     useEffect(() => {
-        const fetchProduct = async () => {
-            try {
-                const response = await fetch(`https://localhost:7207/api/Product/${item.product.id}`);
-                if (!response.ok) {
-                    throw new Error('Network response was not ok');
-                }
-                const data = await response.json();
-                setFormData(data);
-            } catch (error) {
-                console.error('There was a problem fetching product data:', error.message);
-            }
-        };
+
+        if (!item.product.id) {
+            throw new Error('Invalid Id');
+        }
+
         fetchProduct();
+
     }, [item.product.id]);
 
+    // const
+    const API_END_POINT = `https://onboardsite.azurewebsites.net/api/Product/`;
```

Comment:
1. Please check my previous comment about console.logs
2. Please implement the API endpoint that I mentioned above